tests/foreman/cli/test_bootstrap_script.py

```diff
@@ -5,9 +5,8 @@
 
 
 class BootstrapScriptTestCase(CLITestCase):
-    """Test class for bootstrap scipt."""
+    """Test class for bootstrap script."""
 
-    @stubbed()
     @classmethod
     def setUpClass(cls):
         """ create VM for testing """
```

Comment:
As for me ```""" create VM for testing """``` is informative enough. Of course, it isn't actually fully implemented as it is not implemented at all as whole module does. =)